### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by NathanDM**
Big refacto need help for review ;)

### CODE REVIEW

File: front/src/component/app-orga.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+          }
+        };
+      }
+
+      handleTap(e) {
+        //TODO: check with GIUM struct of organisation's ID "Organisation/1"
+        var decodeUrl = this.id.split('/');
+        var orgaId = decodeUrl[decodeUrl.length - 1];
+        e.preventDefault();
+        document.dispatchEvent(new CustomEvent('nav', {
```
Comment (NathanDM) [inline front/src/component/app-orga.html:95]: put it in store ?

File: front/src/component/app-view.html [hunk 2]
```diff
@@ -50,6 +50,14 @@
...
+      static get properties() {
+        return {
+          name: {
+            type: String
+          }
+        }
+      }
       handleTap(e) {
         e.preventDefault();
         document.dispatchEvent(new CustomEvent('nav',{detail: {path: `/view/${this.id}`}}));
```
Comment (NathanDM) [inline front/src/component/app-view.html:63]: listen a store ?

File: front/src/shell/navbar/navbar.html [hunk 3]
```diff
@@ -41,6 +42,8 @@
     </style>
     <div class="header">
       <div class="horizontal-wrapper">
+        <paper-icon-button icon="menu" onclick="drawer.toggle()"></paper-icon-button>
```
Comment (NathanDM) [inline front/src/shell/navbar/navbar.html:45]: remove drawer

File: front/src/shell/page/organisations/organisations.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
+<link rel="import" href="../../../component/app-orga.html">
+<link rel="import" href="../../../../bower_components/app-layout/app-drawer/app-drawer.html">
```
Comment (NathanDM) [inline front/src/shell/page/organisations/organisations.html:2]: remove unused import

File: front/src/component/app-orga.html (hunk 1 above)
Comment (QLevaslot) [reply front/src/component/app-orga.html:95]: I don't think that the navigation is meant to go into the store.

**Pull request merged**